src/schemas/json/unist.json

```
@@ -0,0 +1,71 @@
+{
+  "title": "JSON schema for unist syntax trees",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "Point": {
+      "description": "A point represents one place in a source file.",
+      "type": "object",
+      "properties": {
+        "column": {
+          "description": "The column field (1-indexed integer) represents a column in a source file.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "line": {
+          "description": "The line field (1-indexed integer) represents a line in a source file.",
+          "type": "integer",
+          "minimum": 1
+        },
+        "offset": {
+          "description": "The offset field (0-indexed integer) represents a character in a source file.",
+          "type": "integer",
+          "minimum": 0
+        }
+      },
+      "required": ["column", "line"]
+    },
+    "Position": {
+      "description": "A position represents the location of a node in a source file.",
+      "type": "object",
+      "properties": {
+        "end": {
+          "description": "The end field of Position represents the place of the first character after the parsed source region, whether it exists or not.",
+          "$ref": "#/definitions/Point"
+        },
+        "start": {
+          "description": "The start field of Position represents the place of the first character of the parsed source region.",
+          "$ref": "#/definitions/Point"
+        }
+      },
+      "required": ["end", "start"]
+    }
+  },
+  "type": "object",
+  "description": "Syntactic units in unist syntax trees are called nodes, and implement the Node interface.",
+  "properties": {
+    "type": {
+      "description": "The [type](https://github.com/syntax-tree/unist#type) field is a non-empty string representing the variant of a node. This field can be used to determine the type a node implements.",
```

Just one small suggestion (since markdown isn’t supported in JSON schema descriptions)

```suggestion
      "description": "The type field is a non-empty string representing the variant of a node. This field can be used to determine the type a node implements.",
```

---

Oh I just totally went for it with other descriptions haha. I admit I know that but I find it generally useful in editors to have the link right there in the tooltip. But I'll remove the links.

---

Some editors support the non-standard `markdownDescription` field. You could use this in addition to the `description` field if you like. Personally I tend to omit it as it’s non-standard.

---

There is an unofficial `markdownDescription`  that seems only used by some official VSCode plugins.

---

We got the same idea apprently ^^, I'l use that.